ui/src/app/pages/admin/user/admin-user-egress-bypass.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { Calendar } from 'primereact/calendar';
+import { Column } from 'primereact/column';
+import { DataTable } from 'primereact/datatable';
+
+import { CreateEgressBypassWindowRequest } from 'generated/fetch';
+
+import { Button } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import { TextAreaWithLengthValidationMessage } from 'app/components/inputs';
+import { TooltipTrigger } from 'app/components/popups';
+import { userAdminApi } from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import { formatDate, isDateValid } from 'app/utils/dates';
+const MIN_BYPASS_DESCRIPTION = 10;
+const MAX_BYPASS_DESCRIPTION = 4000;
+
+interface Props {
+  userId: number;
+}
+
+export const AdminUserEgressBypass = (props: Props) => {
+  const [startTime, setStartTime] = useState(null);
```

Comment:
based on how this is used, we should explicitly set the type here with `useState<CalendarValueType>` - this will require other code changes to match this type

Reply:
Done

Reply:
I have a different idea about this change, but that can also wait.